uPortal-core/src/main/java/org/apereo/portal/security/IPermission.java

```diff
@@ -40,6 +40,11 @@
      */
     String PORTLET_SUBSCRIBER_ACTIVITY = "SUBSCRIBE";
     /**
+     * The standard <code>CONFIGURE</code> activity. Allows the user to configure his or her portlet
```

Comment:
Unnecessarily gendered language. Consider instead "Allows the user to configure the portlet"

Reply:
Resolved.